AnimatedWordCloud/Animator/AllocationCalculator/StaticAllocationCalculator/StaticAllocationCalculator.py

```diff
@@ -79,13 +63,15 @@ def allocate(
         words.append(word)
 
     # calculate allocation by selected strategy
-    if strategy == "magnetic":
+    if config.allocation_strategy == "magnetic":
         allocator = MagneticAllocation(
-            image_width, image_height, image_division
+            config.image_width, config.image_height, config.image_division
         )
         return allocator.allocate(words, allocation_before)
     else:
-        raise ValueError("Unknown strategy: {}".format(strategy))
```

Comment:
Do you plan to use any other arguments than the magnetic strategy? If you do not, please skip this review.

Reply:
For example,
config.allocation_strategy: str | function
if config.allocation_strategy == "magnetic":
   // allocate by the magnetic strategy
elif type(config.allocation_strategy) == function:
  // allocate by your own strategy
else:
  raise error

Reply:
This change could be in the future version.

Reply:
There could be another strategy, so I made an option for backward compatibility